README.md

```diff
@@ -60,6 +129,52 @@ unclassified:
     storage: "database"
 ```
 
+Here's an example of how to use it
+
+```
+java -jar java-cli.jar -s http://localhost:8080 -u admin -p <api_token> -f junit-sql-storage-plugin-config.yml
+```
+
+### Accessing Jaeger
+
+You can access the Jaeger UI by going to `http://localhost:16686`, here you can view the performance of the Jenkins server 
+and your plugin changes.
+
+### Accessing the database
+
+You can also query the mysql (or postgres) database by connecting to the `db` container (note: `winpty` is only required when running on Windows git bash)
+
+```
+$ winpty docker exec -it junit-sql-storage-plugin-db-1 mysql --password=root mysql
```

Comment:
winpty can be mentioned separately, shouldn't be in the default example:
```suggestion
$ docker compose exec db mysql --password=root mysql
```

Reply:
Good point, I've removed it from the example.